app/routes/games/list.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  actions: {
+    newGame: function () {
+      // The user must not be anonymous.
+      if (this.get('session.user.isAnonymous')) {
```

Comment:
Updated. It certainly makes the deck list model hook simpler. Also, I imagine it'll make in-game deck wrangling nice and easy.